lms/djangoapps/course_api/views.py

```diff
@@ -123,6 +125,39 @@ def get_object(self):
         )
 
 
+class CourseListUserThrottle(UserRateThrottle):
+    """Limit the number of requests users can make to the course list API."""
+    # The course list endpoint is likely being inefficient with how it's querying
+    # various parts of the code and can take courseware down, it needs to be rate
+    # limited until optimized. LEARNER-5527
+
+    THROTTLE_RATES = {
```

Comment:
We should set the least aggressive rate limit as the default here.

Reply:
Least aggressive meant highest rate limit: 10/20.

Reply:
@robrap @BenjiLee Done.